commons/connector/altoida/upload_altoida_gravity.avsc

```diff
@@ -0,0 +1,13 @@
+{
+  "namespace": "org.radarcns.connector.altoida",
+  "type": "record",
+  "name": "AltoidaGravity",
+  "doc": "Data from 3-axis gravity sensor with meters per square meter (m/s^2) as unit.",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "Timestamp in unix time acquired locally from the device."},
+    {"name": "timeReceived", "type": "double", "doc": "Timestamp in unix time received from Altoida API."},
+    {"name": "x", "type": "float", "doc": "Force of gravity along the x-axis (m/s^2)."},
```

Comment:
is this not with the `g` constant? This is a difference of about `9.81` compared to plain m/s^2.